usagov/vote-gov-drupal
# VOTE-1487 add french string translations and config

## Jira ticket

https://bixal-projects.atlassian.net/browse/VOTE-1487

## Description

Initialize French language in Drupal add string translations, add language date format, and metatag configuration.

## Deployment and testing

### Post-deploy

1. `lando retune`

### QA/Test

1. Visit http://vote-gov.lndo.site/admin/config/regional/language and confirm French in the list of languages.
2. Visit http://vote-gov.lndo.site/admin/config/regional/translate filter by french and translated strings and confirm the string translations imported.
3. Visit http://vote-gov.lndo.site/admin/config/regional/date-time/formats/manage/revision_date/translate/fr/edit and confirm date format for french is set.
4. Visit http://vote-gov.lndo.site/admin/config/search/metatag/global/translate/fr/edit and confirm the metatag configuration for french imported.

## Checklist for the Developer




## Checklist for the Peer Reviewers
